extensions/blocks/membership-button/edit.jsx

```diff
@@ -0,0 +1,337 @@
+/**
+ * External dependencies
+ */
+
+import classnames from 'classnames';
+import SubmitButton from '../../utils/submit-button';
+import apiFetch from '@wordpress/api-fetch';
+import { __ } from '../../utils/i18n';
```

Comment:
Look for https://github.com/Automattic/jetpack/pull/11743 getting merged soon; these i18n functions will need rebasing.